mongodbatlas/processes.go

```diff
@@ -0,0 +1,65 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const processesPath = "groups/%s/processes"
+
+type ProcessesService interface {
```

Comment:
some small nits with all public methods/structs, we try to keep then documented and following and with links to the api docs (https://docs.atlas.mongodb.com/reference/api/processes-get-all/ in this case) you can follow some other implementations as a reference of how we format these docs